test/run

```diff
@@ -95,25 +99,21 @@ testPylibmc() {
 
 testPythonDefault() {
   compile "pythonDefault"
-  assertCaptured "python-3.6.8"
+  assertCaptured $DEFAULT_PYTHON_VERSION
   assertCapturedSuccess
 }
 
 testPython2() {
     compile "python2"
-    assertCaptured "python-2.7.15"
+    assertCaptured $LATEST_27
     assertCapturedSuccess
 }
 
 testPython2_warn() {
     compile "python2_warn"
-    if [[ $STACK = "heroku-18" ]]; then
-      assertCapturedError
-    else
-      assertCaptured "python-2.7.14"
-      assertCaptured "security update!"
-      assertCapturedSuccess
-    fi
+    assertCaptured "python-2.7.15"
```

Comment:
Did you want to add a variable for this too? I suppose it should always warn on `2.7.15` 🤔

Reply:
yeah, I left it as it's now an old version